unity/README.md

```diff
@@ -0,0 +1,25 @@
+# Creating an app package
+1) Build the application as usual within Unity.
+2) In the Visual Studio project, right-click the project in the Solution Explorer and select `Publish -> Create App Packages...`
+
+![Menu](docs/images/publish_screen.png)
+
+3) In the app package window, select `Sideloading`, uncheck the `Enable automatic updates` box, and click `Next`.
+
+![Distribution screen](docs/images/distribution_method.PNG)
+
+4) Ensure the app certificate is signed and click `Next`. The default certificate settings are OK.
+
+![Signing screen](docs/images/signing_method.PNG)
+
+5) Choose the app's output location and ensure the ARM64 architecture is selected. Click `Create` to create the package.
```

Comment:
Does deselecting x86, x64, 32-bit ARM decrease package size / installation time? I am assuming we can just target the one x64 ARM because that's all the HL2 is.

Reply:
Yeah, it's quite a bit faster to only build ARM64. I added a note.